A3-Antistasi/Templates/NewTemplates/CW/CW_AI_CIS.sqf

```diff
@@ -227,18 +227,34 @@ _loadoutData setVariable ["items_medical_medic", ["MEDIC"] call A3A_fnc_itemset_
 _loadoutData setVariable ["items_miscEssentials", [] call A3A_fnc_itemset_miscEssentials];
 
 
-_loadoutData setVariable ["items_squadleader_extras", ["ACE_microDAGR", "ACE_DAGR", "Laserbatteries", "Laserbatteries", "Laserbatteries"]];
+if (A3A_hasACE) then {
+	_loadoutData setVariable ["items_squadleader_extras", ["ACE_microDAGR", "ACE_DAGR", "Laserbatteries", "Laserbatteries", "Laserbatteries"]];
+} else {
+	_loadoutData setVariable ["items_squadleader_extras", ["Laserbatteries", "Laserbatteries", "Laserbatteries"]];
+};
```

Comment:
Is all the copypasting necessary ? Since the loadouts hardly change whether ACE is required, you could do (untested, but I don't see why it shouldn't work):
```suggestion
_loadoutData setVariable ["items_squadleader_extras", [["Laserbatteries", "Laserbatteries", "Laserbatteries"], ["ACE_microDAGR", "ACE_DAGR", "Laserbatteries", "Laserbatteries", "Laserbatteries"]] select A3A_hasACE];
```
Hell, could do **even less** (also untested):
```suggestion
_loadoutData setVariable ["items_squadleader_extras", ["Laserbatteries", "Laserbatteries", "Laserbatteries"] + ([[], ["ACE_microDAGR", "ACE_DAGR"]] select A3A_hasACE)];
```
The point should get across regardless.

Reply:
I'm still learning SQF (coming from a Rust / Swift / Kotlin background), and yeah, this makes more sense, and looks cleaner.

I didn't realize SQF could concatenate arrays. TIL.